test/test_utils.py

```diff
@@ -31,20 +38,40 @@ def test_summarize_report_order(file_name):
 ]
 
 
+example_stock_split_outputs = [
```

Comment:
I'd move this array just above the `test_summarize_report_example_stock_splits` function